forge-game/src/main/java/forge/game/trigger/TriggerHandler.java

```diff
@@ -566,6 +567,19 @@ else if (sa instanceof AbilitySub || !sa.hasParam("Cost") || (sa.getPayCosts() !
         if (regtrig.hasParam("OneOff") && host.isImmutable()) {
             host.getController().getZone(ZoneType.Command).remove(host);
         }
+        if (regtrig.hasParam("LosePerpetual")) {
```

Comment:
Umh...why would you put this in TriggerHandler?

this is something that happens as part of the effect resolving...

Reply:
I just had "OneOff" in my mind from recent memory. 😅 

I'll find a better place for it